src/builtindict.js

```diff
@@ -124,6 +124,7 @@ Sk.setupObjects = function (py3) {
         delete Sk.builtins["StandardError"];
         delete Sk.builtins["unicode"];
         delete Sk.builtins["basestring"];
+        delete Sk.builtins["raw_input"];
```

Comment:
You'll also need to add it back on to builtins for py2 mode.

Some users of skulpt may switch between the py2/py3 setups depending on their user's configuration.

Reply:
Got it, I added this fix in the new commit: https://github.com/skulpt/skulpt/pull/1222/commits/7e207e4cbb286e327b98537db9dab855553cad34
Thank you very much for pointing it out.